src/routes/SingleArticle/index.jsx

```diff
@@ -1,3 +1,34 @@
+import { useState, useEffect } from "react"
+import { useParams } from 'react-router-dom'
+import apiCall from "../../hooks/apiCall";
+
 export const SingleArticle = () => {
+    const { article_id } = useParams();
+    const [article, SetArticle] = useState({})
+    const [isLoading, setIsLoading] = useState(true)
+
+    useEffect(() => {
+        setIsLoading(true)
+        apiCall.get(`articles/${article_id}`).then((response) => {
+            SetArticle(response.data)
+            setIsLoading(false)
+        }).catch(err => console.log(err))
+    }, [])
+
+    if (isLoading) return (<p>Loading</p>)
     
+    return (
+        <main>
+            <article>
+                <img src={article.article_img_url} />
+                <h2>{article.title}</h2>
+                <p>{article.author}</p>
+                <p>{article.body}</p>
+                <p>{article.created_at}</p>
```

Comment:
We also want to see when the articles were created in a human readable format as they should already be sorted by date when they come back from the api